kodi_addon_checker/__main__.py

```diff
@@ -64,10 +66,16 @@ def main():
     parser.add_argument("dir", type=dir_type, nargs="*", help="optional add-on or repo directories")
     parser.add_argument("--branch", choices=choice, required=True,
                         help="Target branch name where the checker will resolve dependencies")
-
+    parser.add_argument("--log_file", help="path to log file")
```

Comment:
let's do a simple `--logfile` instead

Reply:
Or well you can drop this completely.
My original idea was to switch on this and if we get this also print to the console. But this is good enough, if you ask me.

Reply:
Correct, I don't think we have any usecase for that

Reply:
Fixed